README.md

```diff
@@ -7,26 +7,20 @@
 
 A Markdown editor with preview for Sanity Studio. 
 
-Supports Github flavored markdown and image uploads. You can either drag image(s) into the editor or click the bottom bar to bring up a file selector. 
+Supports Github flavored markdown and image uploads. 
+You can either drag image(s) into the editor or click the bottom bar to bring up a file selector. 
 The resulting image URL(s) are inserted with a default width parameter which you can change to your liking using the [Sanity image pipeline parameters](https://www.sanity.io/docs/image-urls).
- 
-### Known issues with the current v3 version
 
-The v2 version used react-mde for editing. The current v3 version @uiw/react-md-editor. This may change before GA.
+The current version is a wrapper around [React SimpleMDE (EasyMDE) Markdown Editor](https://github.com/RIP21/react-simplemde-editor#react-simplemde-easymde-markdown-editor),
+and by extension, [EasyMDE](https://github.com/Ionaru/easy-markdown-editor).
 
-At the moment the v3 version does not have drag-and-drop image upload support.
-You can still add markdown image tags, but you will have to get the image url yourself for now.
+![example.png](./assets/example.png)
 
 ## Installation
+Install `sanity-plugin-markdown` and `easymde` (peer dependency).
 
 ```
-npm install --save sanity-plugin-markdown
-```
-
-or
-
-```
-yarn add sanity-plugin-markdown
+npm install --save sanity-plugin-markdown easymde
```

Comment:
Perhaps set the major version the plugin specifies in its peerDeps as part of the install command?
```suggestion
npm install --save sanity-plugin-markdown easymde@2
```

We don't do this for deps like `react` and `styled-components` as we're testing them daily due to their wide usage in other studio tooling. But for a package like `easymde` maybe it makes sense?

Reply:
We could do this, i guess; but would deviate from react-simplemde which is pulling this